workspace_info.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
```

Comment:
```suggestion
# Copyright 2020 The Kubernetes Authors.
```

Related to the previous review comment: this is a modification of an existing file, so let's preserve history and the copyright date.

Reply:
Removing workspace_info.sh